tf_quant_finance/volatility/bachelier_tf.py

```diff
@@ -0,0 +1,248 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please move the functions to [vanilla_prices](https://github.com/google/tf-quant-finance/blob/master/tf_quant_finance/black_scholes/vanilla_prices.py) module